openvdb/openvdb/tools/FastSweeping.h

```diff
@@ -933,11 +944,33 @@ struct FastSweeping<SdfGridT, ExtValueT>::MinMaxKernel
     {
         if (other.mMin < mMin) mMin = other.mMin;
         if (other.mMax > mMax) mMax = other.mMax;
+        mFltMinExists = other.mFltMinExists || mFltMinExists;
+        mFltMaxExists = other.mFltMaxExists || mFltMaxExists;
     }
 
     SdfValueT mMin, mMax;
+    bool mFltMinExists, mFltMaxExists;
 };// FastSweeping::MinMaxKernel
 
+/// Private class of FastSweeping to prune sdf value that is near float max or
+/// float min.
+template <typename SdfGridT, typename ExtValueT>
+struct FastSweeping<SdfGridT, ExtValueT>::PruneMinMaxFltKernel {
+
+  PruneMinMaxFltKernel(SdfValueT min, SdfValueT max) : mMin(min), mMax(max) {}
+
+  inline void operator()(const typename SdfGridT::ValueAllIter &iter) const {
+    if (std::abs(*iter + std::numeric_limits<SdfValueT>::max()) < 1.0e-4f) {
+      iter.setValue(mMin);
+    }
+    if (std::abs(*iter - std::numeric_limits<SdfValueT>::max()) < 1.0e-4f) {
```

Comment:
these supposed to be equality too?

Reply:
Thanks for catching this. I have fixed it in a new commit.